tests/visual/listbox.html

```diff
@@ -0,0 +1,75 @@
+                        <a href="#" class="k-button k-button-icon" data-command="moveUp" title="Move Up" aria-label="Move Up"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-up"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="moveDown" title="Move Down" aria-label="Move Down"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-down"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferTo" title="Transfer To" aria-label="Transfer To"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-right"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferFrom" title="Transfer From" aria-label="Transfer From"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-left"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferAllTo" title="Transfer All To" aria-label="Transfer All To"
+                            role="button">
+                            <span class="k-icon k-i-arrow-double-60-right"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferAllFrom" title="Transfer All From" aria-label="Transfer All From"
+                            role="button">
+                            <span class="k-icon k-i-arrow-double-60-left"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="remove" title="Delete" aria-label="Delete" role="button">
+                            <span class="k-icon k-i-x"></span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="k-list-scroller k-selectable" data-role="selectable">
+                <ul class="k-reset k-list" role="listbox" tabindex="0">
+                    <li class="k-item" role="option" aria-selected="false" data-uid="6bd99714-38a2-4053-80c4-48494c8f6afb">Item</li>
+                    <li class="k-item k-state-selected" role="option" aria-selected="false" data-uid="19d76c92-35e9-4e0a-9135-f03d49dfe159">Selected</li>
+                    <li class="k-item k-state-focused" role="option" aria-selected="false" data-uid="a5a173ed-8c8e-488c-b297-faa2f68518f0">Focused</li>
```

Comment:
The test does not show hovered, hover-selected and disabled item.

Additionally, there is not drag clue.